automation/check-patch.rhel8.packages

```diff
@@ -0,0 +1,7 @@
+git
+docker
+jq
+ansible
+libosinfo
+expect
+python-gobject
```

Comment:
what is python-gojbect needed for?

Reply:
I think it is needed for communication between osinfo-db and script which creates templates.